src/main/DateTime.java

```diff
@@ -0,0 +1,11 @@
+package main;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class DateTime {
+    public static final DateTimeFormatter dmy = DateTimeFormatter.ofPattern("dd MMM yyyy");
```

Comment:
Perhaps you could use a more descriptive name.
```suggestion
    public static final DateTimeFormatter dateFormat = DateTimeFormatter.ofPattern("dd MMM yyyy");
```